docs/pbc/all/search/algolia.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Algolia
+description: Algolia empowers Builders with Search and Recommendation services to create world-class digital experiences.
+---
+
+Spryker is shipped with [Elasticsearch]((https://www.elastic.co/elasticsearch/)) as the default search engine. However, you can replace it with [Algolia](https://www.algolia.com/).
+
+The Algolia search engine stands out from others due to its performance. With the Algolia app, your users can conduct advanced search of active concrete products in your store. 
+
+{% info_block infoBox "Product reviews" %}
+
+Algolia searches in product reviews as well. However, product reviews are represented in product attributes in the Algolia search results.
```

Comment:
@lenadoc 
If you mean to search by content (text) of product reviews - it's not working.
if you mean the ability to filter products by average rating (only 5 stars or between 3.5 and 4.5) - it will work